daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/scenario/ScenarioLedger.scala

```diff
@@ -181,7 +183,7 @@ object ScenarioLedger {
     /** 'True' if the given 'View' contains the given 'Node'. */
     def visibleIn(view: View): Boolean = view match {
       case OperatorView => true
-      case ParticipantView(party) => disclosures contains party
+      case ParticipantView(parties) => parties.any(disclosures.contains(_))
```

Comment:
Is there nothing similar to
```suggestion
      case ParticipantView(parties) => parties.intersects(disclosures)
```